### PR TITLE
KAFKA-147: copy existing databases and collections by regex

### PR DESCRIPTION
**Pull request by ortex**
https://jira.mongodb.org/browse/KAFKA-147

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,247 @@
+root = true
```
Comment (ortex) [inline .editorconfig:1]: Most parameters are default, except:

* indent_size
* tab_width
* ij_continuation_indent_size
* ij_java_names_count_to_use_import_on_demand
* ij_java_class_count_to_use_import_on_demand

**Comment by ortex** [comment]
Hi @rozza 
Please, have a look

**Comment by rozza** [comment]
Hi @ortex, I think this looks good, however, rather than adding two configurations a single namespace regex config would suffice.  I converted this PR to use a single config: https://github.com/rozza/mongo-kafka/tree/k147 - let me know what you think.

**Comment by ortex** [comment]
> Hi @ortex, I think this looks good, however, rather than adding two configurations a single namespace regex config would suffice. I converted this PR to use a single config: https://github.com/rozza/mongo-kafka/tree/k147 - let me know what you think.

I like your idea with a single `copy.existing.namespace.regex` config param.

**Comment by ortex** [comment]
I guess `COPY_EXISTING_NAMESPACE_REGEX_DOC` should be a little more detailed. For example:

```
Regex to define from which existing namespaces data should be copied from. Regex will be run against the namespace's name which is the database name and the collection name, separated by a period e.g. `database.collection`
```


And what do we do next? 
I should cherry-pick your commit or you will create another PR?

**Comment by rozza** [comment]
Great news, I'll update the docs, then will merge before 1.3.0 code friends

**Comment by rozza** [comment]
Closing this PR - but it is the basis for the new feature added in KAFKA-147

**Comment by ortex** [comment]
Great, thanks. 
Do you have a release date for 1.3.0?

**Comment by rozza** [comment]
Oct 1st is the official plan. There aren't many tickets left so the build may be in maven by the end of this week.

**Pull request closed without merging**